applications/nrf_desktop/src/modules/led_stream.c

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <misc/byteorder.h>
+
+#define MODULE led_stream
+#include "module_state_event.h"
+
+#include <logging/log.h>
+
+#include "config_event.h"
+#include "led_event.h"
+
+#define STREAM_LED_ID 0
+#define INCOMING_LED_COLOR_COUNT 3
+#define STREAM_DATA_SIZE 7
+
+LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_LED_STREAM_LOG_LEVEL);
+
+
+static struct led_effect first_led_stream_effect = {
```

Comment:
please use array instead of two variables. it is easier to extend + switching between elements is easier than providing different names

Reply:
done